ModularTegustation/tegu_mobs/the_claw.dm

```diff
@@ -11,7 +11,7 @@
 	icon_living = "claw"
 	icon_dead = "claw_dead"
 	icon = 'ModularTegustation/Teguicons/48x48.dmi'
-	faction = list("Head")
+	faction = list("hostile", "Head")
```

Comment:
Why? Not necessarily a negatively charged question just wondering the intent here

Reply:
I don't want Claw to be fighting random abnormalities. It is here to fight the players, not random things.